js/rpg_scenes/Scene_Base.js

```diff
@@ -31,6 +32,7 @@ Scene_Base.prototype.isReady = function() {
 
 Scene_Base.prototype.start = function() {
     this._active = true;
+    ImageManager.setDefaultReservationId(this._imageReservationId);
```

Comment:
Hm, is this timing okay?
Reserved bitmaps during `create()` and `start()` seem to be reserved forever...